src/gcp_scanner/credsdb.py

```diff
@@ -224,8 +224,11 @@ def get_access_tokens_dict(path_to_creds_db: str) -> Dict[str, str]:
       token = row[1]
       expiration_date = row[2]
 
-      token_time_obj = datetime.datetime.strptime(expiration_date,
-                                                  "%Y-%m-%d %H:%M:%S.%f")
+      for fmt in ["%Y-%m-%d %H:%M:%S.%f", "%Y-%m-%d %H:%M:%S"]:
```

Comment:
I'd just omit .%f to avoid any errors and exceptions at all and make code as simple as possible.

Reply:
I omitted the .%f to avoid the error